packages/ui-react/src/components/TileDock/TileDock.tsx

```diff
@@ -263,13 +268,17 @@ function TileDock<T>({
             />
           ) : null}
           {tileList.map((tile: Tile<T>, listIndex) => {
+            const posInSet = items.findIndex((item) => item === tile.item); // TODO optimize this for performances
             const isInView = !isMultiPage || (listIndex > tilesToShow - slideOffset && listIndex < tilesToShow * 2 + 1 - slideOffset);
 
             return (
               <li
                 key={tile.key}
+                aria-setsize={items.length}
+                aria-posinset={posInSet + 1}
                 aria-hidden={!isInView}
                 className={classNames({ [styles.notInView]: !isInView })}
+                tabIndex={-1}
```

Comment:
This is a requirement to allow the focus to be set programmatically to the list-item

Reply:
But do we need to focus the list items? I can imagine we want to focus cards inside the list instead. 🤔

Reply:
I also tried out programmatically setting the focus to the card (the button). But then it does not read the position in the list (`aria-setsize` and `aria-postinset`) which also contains valueable information.